docs/commands/link.md

```diff
@@ -20,6 +20,7 @@ netlify link
 
 - `filter` (*string*) - For monorepos, specify the name of the application to run the command in
 - `git-remote-name` (*string*) - Name of Git remote to use. e.g. "origin"
+- `git-remote-url` (*string*) - URL of the repository (or Github `owner/repo`) to link to
```

Comment:
It was one extra line of code to make this an option for `netlify link` as well, so I went ahead and added it